Add role to upgrade postgresql on EL8 (#21)
theforeman/foreman-operations-collection: roles/postgresql_upgrade/README.md

theforeman.operations.postgresql_upgrade
========================================

> Upgrades Postgresql for a Foreman install.

Reviewer: ```suggestion
Upgrades PostgreSQL for a Foreman install.
```

Reviewer: I think this also should point out that this is only for EL8 systems with module-based PSQL, which will update 10 to 12, not any arbitrary versions.

> Upgrades PostgreSQL for a Foreman install on EL8 only.

Reviewer: ```suggestion
Upgrade Foreman's PostgreSQL from 10 to 12 on EL8.
```

> Role Variables
> --------------
> 

Reviewer: ```suggestion
```
